webcompat/static/js/lib/bugform.js

```diff
@@ -491,8 +499,13 @@ function BugForm() {
       img_url,
       ")"
     ].join("");
+    var delimeter = "[![Screenshot Description](";
     this.descField.val(function(idx, value) {
-      return value + "\n\n" + imageURL;
+      if (value.includes(delimeter)) {
```

Comment:
@miketaylr will do. just to let you know that this PR is not ready to review. I only pushed the branch to have a point of reference to discuss with Ola about how to begin to approach the problem.